Clubs_level_1_char_sheet.html

```diff
@@ -0,0 +1,351 @@
+
+<html>
```

Comment:
The embedding on this file might be easier to see if we ran `prettier` on it. Like `black` but for other file types